packages/botonic-plugin-ner/src/types.ts

```diff
@@ -1,10 +1,11 @@
 import { NerConfig } from '@botonic/nlp/dist/tasks/ner/storage/types'
 import { Locale } from '@botonic/nlp/dist/types'
 import { LayersModel } from '@tensorflow/tfjs'
+import { AxiosPromise } from 'axios'
 
-export type ModelInfo = {
-  config: NerConfig
-  model: LayersModel
+export type ModelInfoPromise = {
```

Comment:
since it's not actually a Promise, I think it makes sense to call it st like ModelInfoPromises to avoid that developers do await on the whole object.
I'm not sure if we should change the function that creates it to return instead of `Promise<{config:NerConfig, model: LayersModel}`. But we can do it in the future.